src/styles/override/_alerts.scss

```diff
@@ -2,6 +2,10 @@
   @include alert-variant($primary, $primary, $white);
 }
 
+.alert-danger {
+  @include alert-variant($light-pink, $light-pink, $white);
```

Comment:
Why? If we have different color scheme for alerts, then may we change their variables in variables.scss instead?

Reply:
"alert-danger" variant by default uses $danger color, but we shoudn't change it because this $danger color is used in other places. So i just replaced color for this particular variant.

Reply:
Looks like bootstrap 5 has no alert variables in comparison to bootstrap 4.